src/pfe/portal/modules/ExtensionList.js

```diff
@@ -23,6 +23,8 @@ const log = new Logger(__filename);
 const extensionsDir = '/extensions';
 const extensionsPattern = /^(\S+)-(\d+\.\d+\.\d+)\.zip$/; // e.g. extension-name-0.0.1.zip
 const suffixOld = '__old';
+const odoExtensionName = "codewind-odo-extension";
+const odoBinarySource = "https://github.com/openshift/odo/releases/latest/download/odo-linux-amd64"
```

Comment:
Is the hardcoding of odo stuff in this file temporary?

Reply:
We may or may not update/move this depends on the design/ODO CLI release. (eg. `https://github.com/openshift/odo/releases/latest/download/odo-linux-amd64` links to the latest stable ODO CLI, we don't need to change it for now, but may do some improvements for the installer)

Reply:
But I mean, we should not have "ODO" specific things hardcoded in Codwind.  I think @elsony there was an issue opened to further clean up hardcoding of extension specific things, so I was trying to confirm if this is also one of the items to be cleaned up later.

Do you guys have a link to that issue by any chance?